docs/content/preview/yugabyte-platform/troubleshoot/install-upgrade-issues/kubernetes.md

```diff
@@ -0,0 +1,309 @@
+  ##   Warning  Failed                  22s (x3 over 72s)   kubelet                  Error: ErrImagePull
+  ##   Normal   BackOff                 7s (x3 over 72s)    kubelet                  Back-off pulling image "quay.io/yugabyte/yugaware:2.16.0.0-b90"
+  ##   Warning  Failed                  7s (x3 over 72s)    kubelet                  Error: ImagePullBackOff
+  ```
+
+**Resolution**
+
+- In case of **Bad pull secret / no pull secret / Bad pull secret name** - You need a pull secret to fetch the images from Yugabyte Quay.io and ensure you have applied the same in the namespace, which will use to install YBA. It will default look for the following named `yugabyte-k8s-pull-secret` secret. For more information, see [values.yaml](https://github.com/yugabyte/charts/blob/24a8dcf3a4c33153477e3e3ba82f9f4b6e2967a5/stable/yugaware/values.yaml#L16).
+- For **Unable to pull image** - Ensure the kubernetes nodes have connectivity to Quay.io, or you should have images in the local registry. For more information, see [pull and push yugabytedb docker images to private container registry](https://docs.yugabyte.com/preview/yugabyte-platform/install-yugabyte-platform/prerequisites/#pull-and-push-yugabytedb-docker-images-to-private-container-registry).
+
+
+## Pods are in CrashLoopBackOff
+
+**Symptom**
+
+There can be multiple reasons behind `CrashLoopBackOff`. YBA pod can crash due to some internal application error.
+
+```sh
+## NAME                             READY   STATUS             RESTARTS   AGE
+## yugabyte-platform-1-yugaware-0   3/4     CrashLoopBackOff   2          4d14h
+```
+
+**Debug**
+
+You can use `kubectl` to analyze the situation. We need to check YBA logs to understand the failure.
+
+- Describe the YBA pod facing `CrashLoopBackOff`. Check for `Events`.
+
+  ```sh
+  kubectl describe pods <YBA-POD-NAME> -n <NAMESPACE>
+  ```
+
+- Check the logs for particular container
+
+  ```sh
+  ## yugabytedb anywhere
+  kubectl logs <YBA-POD-NAME> -n <NAMESPACE> -c yugaware
+
+  ## postgres
+  kubectl logs <YBA-POD-NAME> -n <NAMESPACE> -c postgres
+  ```
+
+## CORS error while accessing YBA through LB
+
+You might experience the issue while accessing the YugabyteDB Anywhere through a load balancer. You can define the Cross-Origin Resource Sharing (CORS) domain configuration by setting the [additionAllowedCorsOrigins](https://github.com/yugabyte/charts/blob/master/stable/yugaware/values.yaml#L66) value to the new domain involved. For example, you would add the following to the appropriate Helm command.
+
+  ```properties
+   --set additionAllowedCorsOrigins:'https://mylbdomain'
+  ```
+
+## Load Balancer + EKS
+
+If the load balancer is not working as intended for internet facing, see [EKS + Load Balancer](https://docs.yugabyte.com/stable/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes/).
```

Comment:
This link doesn't have anything related to eks? Maybe we just need to rebase?

Also, should we say "if an EKS internet-facing load balancer is unable to be created, see ... "?